opensearch-project/OpenSearch-Dashboards
# [Multiple Datasource] Add empty icon and fix empty state handling

### Description
This change adds empty database icon to handle when there is no clusters to show, and I also changed the toast message and popover based on requirements.

### Issues Resolved
Fixes https://github.com/opensearch-project/OpenSearch-Dashboards/issues/6565

## Screenshot

https://github.com/opensearch-project/OpenSearch-Dashboards/assets/17714150/95d540e8-5bc3-4576-9430-d6a42aec972b



## Testing the changes
The following steps were performed in the recording:
1. go to multi select example, and see toasts about empty state, click on icon will show popover, the link in the popover goes to data source management page
2. go to single select example, and see toasts about empty state, click on icon will show popover, the link in the popover goes to data source management page
3. go to single readonly example, and see toasts about empty state, click on icon will show popover, the link in the popover goes to data source management page
4. go to aggregated list all example, and see toasts about empty state, click on icon will show popover, the link in the popover goes to data source management page


## Changelog
- skip


### Check List
